boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270.dtsi

```diff
@@ -58,3 +59,67 @@
 &sar_adc1 {
 	vref-mv = <1800>;
 };
+
+&qspi0 {
+	pinctrl-0 = <&qspi0_default>;
+	pinctrl-names = "default";
+	data-rate = "DDR";
+	column-space = <3>;
+	word-addressable;
+	hold-time-2x;
+	a-dll-freq-enable;
+	a-dll-ref-counter = <2>;
+	a-dll-resolution = <2>;
+	a-dll-coarse-delay = <2>;
+	a-dll-fine-delay = <2>;
+	a-dll-mode = "AUTO UPDATE";
+	a-rx-clock-source = "EXTERNAL DQS";
+	ahb-buffers-masters = <0 1 2 3>;
+	ahb-buffers-sizes = <256 256 256 256>;
+	ahb-buffers-all-masters;
+	status = "okay";
+
+	sfp {
+		compatible = "secure-flash-protection";
```

Comment:
Is this compatible needed? I don't see it defined in tree.

Reply:
It is needed for building this device tree node.

Reply:
I'm not sure I follow- if we don't have a compatible here, does the support no longer work?

Reply:
I previously added the parent node 'sfp' to address a reviewer's [comment](https://github.com/zephyrproject-rtos/zephyr/pull/78073#discussion_r1798966791). Its binding is not defined but it is required for this node. By adding the 'secure-flash-protection' compatibility, we can differentiate it from other nodes. Without this, Zephyr would build this node as 'nxp,s32-qspi-hyperflash', which requires non-existent properties and causes build failures.

Reply:
I updated to remove this compatible